solidity/utils/README.md

```diff
@@ -114,8 +114,7 @@ Here is an example of the initial configuration file which should be provided to
         }
     ],
     "networkSettingsParams": {

```

Comment:
Should remove the entire `networkSettingsParams` field (not just the `networkFee` field).

Reply:
Done